examples/kuba/students.py

```diff
@@ -0,0 +1,81 @@
+# Dictionary that represents students from last class.
+# Key: unique id, Value: (name, semester, list of favorite OS's)
+students = {
+    0: ("Adam", 7, ["Linux", "Windows 10", "Mac OS"]),
+    1: ("Konrad", 3, ["Linux"]),
+    2: ("Monika", 3, ["Linux", "Windows"]),
+    3: ("Piotrek", 3, ["Linux"]),
+    4: ("Kuba", 3, ["Mac OS"]),
+    5: ("Krzysiek", 3, ["Linux"]),
+    6: ("Krzysiek", 3, ["Windows 10"]),
+    7: ("Adam", 1, ["Windows 10"]),
+    8: ("Marcel", 1, ["Windows 10"]),
+    9: ("Paulina", 1, ["Windows 10"]),
+    10: ("Sebastian", 1, ["Windows 10"])
+}
+# students_count(students) - number of students in class
+print(len(students))
+
+# was_present(students, name) returns True only if someone of given name was on last class.
+def was_present(students, name):
+    for klucz, abbrev in students.items():
+        while abbrev[0] != name:
+            return False
+        return True
+
+print(was_present(students, "Daniel"))
+
+#for klucz, abbrev in students.items():
+ #   print((klucz, abbrev))
+
+# unique_was_present(students, unique_id) returns True only if someone of given unique_id was on last class.
+def unique_was_present(students, unique_id):
+    if unique_id in students:

```

Comment:
Jest ok. A co myślisz o `return unique_id in students` ? To tylko jedna linijka :)